### PR TITLE
Update the external bits like imagestreams, templates etc.

### PR DESCRIPTION
**Pull request by mkumatag**
Updated the external example bits via script: `./hack/update-external-examples.sh` and `hack/update-generated-bindata.sh`, this will help other archs like ppc, s390x to run the test smoother with newly updated images.

### CODE REVIEW

**Comment by bparees** [comment]
@mkumatag the updated content is from running `hack/update-external-examples.sh` ?

File: examples/db-templates/mysql-ephemeral-template.json [hunk 1]
```diff
@@ -267,11 +267,11 @@
             "value": "sampledb"
         },
         {
-            "description": "Version of MySQL image to be used (5.7, or latest).",
+            "description": "Version of MySQL image to be used (8.0, or latest).",
             "displayName": "Version of MySQL Image",
             "name": "MYSQL_VERSION",
             "required": true,
-            "value": "5.7"
+            "value": "8.0"
```
Comment (gabemontero) [inline examples/db-templates/mysql-ephemeral-template.json]: there is a known issue with running our templates with mysql:8 ... the SCL team is suppose to be working it

in the interim, you should keep this at 5.7

File: examples/db-templates/mysql-persistent-template.json [hunk 2]
```diff
@@ -272,11 +272,11 @@
             "value": "1Gi"
         },
         {
-            "description": "Version of MySQL image to be used (5.7, or latest).",
+            "description": "Version of MySQL image to be used (8.0, or latest).",
             "displayName": "Version of MySQL Image",
             "name": "MYSQL_VERSION",
             "required": true,
-            "value": "5.7"
+            "value": "8.0"
```
Comment (gabemontero) [inline examples/db-templates/mysql-persistent-template.json]: there is a known issue with running our templates with mysql:8 ... the SCL team is suppose to be working it

in the interim, you should keep this at 5.7

**Comment by mkumatag** [comment]
> @mkumatag the updated content is from running `hack/update-external-examples.sh` ?

yes

File: examples/db-templates/mysql-persistent-template.json (hunk 2 above)
Comment (mkumatag) [reply examples/db-templates/mysql-persistent-template.json]: done!

File: examples/db-templates/mysql-ephemeral-template.json (hunk 1 above)
Comment (mkumatag) [reply examples/db-templates/mysql-ephemeral-template.json]: done!

Comment (gabemontero) [reply examples/db-templates/mysql-ephemeral-template.json]: fyi ... tracking item for  SCL issue is https://bugzilla.redhat.com/show_bug.cgi?id=1793116

**Comment by mkumatag** [comment]
I hate this error:
```
could not wait for RPM repo server to deploy: could not wait for deployment: the deployment config ci-op-5h8vml3x/rpm-repo failed with status "ProgressDeadlineExceeded"
```
@gabemontero I'm seeing this error for 2 days now, is this very common in openshift PR tests!

**Comment by gabemontero** [comment]
On Wed, Jan 22, 2020 at 12:06 PM Manjunath Kumatagi <
notifications@github.com> wrote:

> I hate this error:
>
> could not wait for RPM repo server to deploy: could not wait for deployment: the deployment config ci-op-5h8vml3x/rpm-repo failed with status "ProgressDeadlineExceeded"
>
> @gabemontero <https://github.com/gabemontero> I'm seeing this error for 2
> days now, is this very common in openshift PR tests!
>

Our CI has been in bad shape on multiple fronts for the last week or so.
We are all affected.

**Pull request closed without merging**